Feature: wcmatch for automatic build triggering (#1319)
codemagic-ci-cd/codemagic-docs: content/flutter-running-builds/starting-builds-automatically.md

> ## Tracking specific branches and tags

Reviewer: ```suggestion
## Tracking specific branches or tags
```

## Tracking specific branches or tags

Instead of watching all branches and tags, you can limit automatic build triggering to branches or tags whose name matches a specific pattern.

The branches or tags tracked for building are selected by entering one or more patterns and including or excluding the matching branches or tags.

> Note that you can either enter the exact name of the branch or tag to select it or use the wildcard symbols to select more than one branch with one pattern. Click **Show pattern examples** in the UI for tips. Additionally, please refer to [Wildcard Match Documentation](https://facelessuser.github.io/wcmatch/fnmatch/) for more advanced matching patterns.

Reviewer: ```suggestion
Note that you can either enter the exact name of the branch or tag or use wildcard symbols to match multiple branches or tags. Click **Show pattern examples** in the UI for tips. Additionally, please refer to [Wildcard Match Documentation](https://facelessuser.github.io/wcmatch/fnmatch/) for more advanced matching patterns.
```
